fiat-bedrock2/src/curve25519_64.c

```diff
@@ -930,14 +971,19 @@ void fiat_25519_from_bytes(uintptr_t out0, uintptr_t in0) {
   return;
 }
 
+void fiat_25519_from_bytes(uint64_t out1[5], const uint64_t arg1[32]) {
```

Comment:
Somehow we'll need to figure out how to know that this is supposed to be `uint8_t` ... :-(
```suggestion
void fiat_25519_from_bytes(uint64_t out1[5], const uint8_t arg1[32]) {
```